RMv4_Recipely/RMv4_Recipely/FullRecipe.swift

```diff
@@ -0,0 +1,38 @@
+// FullRecipe.swift
+// Copyright © RoadMap. All rights reserved.
+
+import Foundation
+
+/// Модель полного описания рецепта
+struct FullRecipe {
+    // Название рецепта
```

Comment:
```suggestion
    /// Название рецепта
```
/// - дока
// - комментарий